torchaudio/datasets/AudioMNIST.py

```diff
@@ -0,0 +1,227 @@
+from __future__ import print_function
+from glob import glob
+import torch.utils.data as data
+import os
+import os.path
+import errno
+import torch
+import torchaudio
+import json
+import random
+import math
+import urllib
+import zipfile
+from tqdm import tqdm
+
+
+class AudioMNIST(data.Dataset):
+    """ `AudioMNIST <https://github.com/soerenab/AudioMNIST/>`_ Dataset.
+
+    Args:
+        root (string): Root directory of dataset where ``processed/training.pt``
+            and ``processed/test.pt`` exist.
+        split (string): The dataset has 3 different splits: ``digit``,
+            ``speaker``, and ``gender``. This argument specifies which one to use.
+        train (bool, optonal): If True, creates dataset from ``training.pt``,
+            otherwise from ``test.pt``.
+        download (bool, optional): If true, downloads the dataset from the internet and
+            puts it in the root directory. If dataset is already downloaded, it is not
+            downloaded again.
+        transform (callable, optional): A function/transform that takes in an audio
+            tensor and returns a transformed version. E.g. ``transforms.Scale``
+        target_transform (callable, optional) A function/transform that takes in the
+            target and transforms it.
+        dev_mode(bool, optional): if true, clean up is not performed on downloaded
+            files. Useful to keep raw audio.
+    """
+    url = 'https://github.com/soerenab/AudioMNIST/archive/master.zip'
+    splits = ('digit', 'speaker', 'gender')
+    raw_folder = 'raw'
+    processed_folder = 'processed'
+
+    def __init__(self, root, split, train=True, transform=None, target_transform=None, download=False):
+        self.root = os.path.expanduser(root)
+        self.transform = transform
+        self.target_transform = target_transform
+        self.train = train  # training set or test set
+
+        if download:
+            self.download()
+
+        # if not self._check_exists():
```

Comment:
why is this commented?

Reply:
Sorry, it shouldn't be commented out - I'm not sure why it is.